Library/Formula/rack.rb

```diff
@@ -0,0 +1,27 @@
+class Rack < Formula
+  desc "CLI for Rackspace"
+  homepage "https://github.com/rackspace/rack"
+  url "https://github.com/rackspace/rack.git",
+      :tag => "1.0.1",
+      :revision => "71a8d7c80b3652b4dd39e683bc423b8a542b0167"
+  head "https://github.com/rackspace/rack.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["TRAVIS_TAG"] = @tag
+
+    rackpath = buildpath/"src/github.com/rackspace/rack"
+    rackpath.install (Dir["{*,.*}"] - [".", ".."])
+
+    cd rackpath do
+      system "script/build", "rack"
+      bin.install "rack"
+    end
+  end
+
+  test do
+    system "#{bin}/rack"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!